pip/qsharp/_qsharp.py

```diff
@@ -273,10 +273,46 @@ def set_classical_seed(seed: Optional[int]) -> None:
     """
     get_interpreter().set_classical_seed(seed)
 
+class StateDump:
```

Comment:
I wonder if you'd get to avoid duplicating methods if you made this a subclass of `StateDumpData` instead of having it as a member?

Reply:
Hmm... I tried this in the hopes that it would work, but got some arcane Python errors. Given that it won't save us too much in the implementation, I think we should leave this as-is for now.